src/components/search/search.spec.tsx

```diff
@@ -545,66 +523,26 @@ describe("Search", () => {
 
   describe("tab index should be set on clear button", () => {
     it("when input field is not empty", () => {
-      wrapper = renderWrapper(
-        {
-          defaultValue: "Bar",
-          id: "Search",
-          name: "Search",
-        },
-        mount
-      );
+      wrapper = renderSearch({
+        defaultValue: "Bar",
+        id: "Search",
+        name: "Search",
+      });
       const input = wrapper.find(TextBox);
       expect(input.prop("iconTabIndex")).toEqual(0);
     });
 
     it("when input field is empty", () => {
-      wrapper = renderWrapper(
-        {
-          defaultValue: "",
-          id: "Search",
-          name: "Search",
-        },
-        mount
-      );
+      wrapper = renderSearch({
+        defaultValue: "",
+        id: "Search",
+        name: "Search",
+      });
       const input = wrapper.find(TextBox);
       expect(input.prop("iconTabIndex")).toEqual(-1);
     });
   });
 
-  describe("tab index should be set on search button", () => {
```

Comment:
Is there a good reason to simply remove these tests? (Happy to accept it if there is!)

Reply:
to further explain: tabIndex on the button is changed to `-1`, when the search input has no text. The search button is not rendered at all when the input has no text and it's not focused, so there is no need to pass that prop down, as the button will be removed as soon as user move focus out of the input.